### PR TITLE
Update and use mrmime

### PR DESCRIPTION
**Pull request by joeosburn**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -262,9 +252,9 @@
       "dev": true
     },
     "callsites": {
-      "version": "2.0.0",
-      "resolved": "https://registry.npmjs.org/callsites/-/callsites-2.0.0.tgz",
-      "integrity": "sha1-BuuE8A7qQT2oav/vrL/7Ngk7PFA="
+      "version": "4.0.0",
+      "resolved": "https://registry.npmjs.org/callsites/-/callsites-4.0.0.tgz",
+      "integrity": "sha512-y3jRROutgpKdz5vzEhWM34TidDU8vkJppF8dszITeb1PQmSqV3DTxyV8G/lyO/DNvtE1YTedehmw9MPZsCBHxQ=="
```
Comment (shanebo) [inline package-lock.json]: @joeosburn this will break it

Comment (joeosburn) [reply package-lock.json]: That was from the force update. The final version of the package lock uses callsites 3

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "@dylan/coerce": "~0.2.0",
-    "@dylan/engines": "github:shanebo/engines#0.1.0",
+    "@dylan/engines": "^0.1.1",
```
Comment (shanebo) [inline package.json:19]: @joeosburn shouldn't this be a `~`?

Comment (joeosburn) [reply package.json]: Yep, the npm install changed it and I missed it. It's fixed now.

File: package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "@dylan/coerce": "~0.2.0",
     "@dylan/engines": "github:shanebo/engines#0.1.0",
     "beard": "~0.8.1",
-    "mime": "2.3.1",
```
Comment (shanebo) [inline package.json:21]: Seems fitting to squash this commit with the commit where you actually utilize mrmime

File: package-lock.json [hunk 4]
```diff
@@ -11,9 +11,9 @@
       "dev": true
     },
     "@babel/runtime-corejs2": {
-      "version": "7.15.3",
-      "resolved": "https://registry.npmjs.org/@babel/runtime-corejs2/-/runtime-corejs2-7.15.3.tgz",
-      "integrity": "sha512-iG7ypZmrdoKP1ckFurS8z97TR+Bqd6KaDsLQ9DiC/Rdxmrvy1nsCDlgfLNKfalbg9sFWdmIdNf+Hg+19XysSFg==",
+      "version": "7.16.0",
+      "resolved": "https://registry.npmjs.org/@babel/runtime-corejs2/-/runtime-corejs2-7.16.0.tgz",
+      "integrity": "sha512-PICpXRloQQ7AtgN+8xYbOxaopQ1vMB9RV0tsVe5ikx5/6y7iO8tYQiyAf3hIYxwz8AohnSXBuXorBacFOs21xg==",
```
Comment (shanebo) [inline package-lock.json:16]: This seems to be an artifact leftover from the npm update --force. Seems safest to amend this commit, delete the package lock and npm install to make sure this is accurate.

Comment (joeosburn) [reply package-lock.json:16]: what commit is this from? I can't tell from your comment

Comment (joeosburn) [reply package-lock.json:16]: done

File: .github/workflows/node.js.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+name: Dylan CI
```
Comment (shanebo) [inline .github/workflows/node.js.yml:1]: I'd put this commit before you bump the version

**Review by shanebo — APPROVED** [review]

**Pull request merged**